molecule/centos/molecule.yml

```diff
@@ -0,0 +1,84 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+scenario:
+  name: centos
+lint:
+  name: yamllint
+platforms:
+  - name: centos-icinga2-master
+    groups:
+      - icinga2master
+      - icingaweb2
+    image: centos:8
+    command: /sbin/init
+    privileged: True
+  - name: centos-icinga2-satellite
+    groups:
+      - icinga2satellite
+    image: centos:7
+    command: /sbin/init
+    privileged: True
+  - name: centos-icinga2-satclient
+    groups:
+      - icinga2client
+      - icinga2satclient
+    image: centos:8
+    command: /sbin/init
+    privileged: True
+  - name: centos-icinga2-std-client
+    groups:
+      - icinga2client
+      - icinga2stdclient
+    image: centos:6
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      callback_whitelist: profile_tasks
+      jinja2_extensions: jinja2.ext.do
```

Comment:
I don't see a need for this extension. You would only need it if you use "do" and "set" in jinja templates.